.github/labeler.yml

```diff
@@ -0,0 +1,26 @@
+# Label to be added when the PR is updated:
+"Awaiting Review":
+  - "**/*"
+
+# Labels to be removed when the PR is updated:
+"Reviewed: Ready to Merge": []
+"Reviewed: Action Needed": []
+
+# Labels based on which files are modified:
+"Gem Updates":
+  - "Gemfile"
+  - "Gemfile.lock"
+
+"Has Migrations":
+  - "db/migrate/**/*"
+
+"Has Production Config Changes":
```

Comment:
I think you can leave this out. We only need this label if we expect the added values to be overridden in local.yml on staging/production (e.g. Zoho configs), in order words follow-up actions for our sysadmins. Most of the time we don't.